packages/website/src/clientModules.js

```diff
@@ -1,3 +1,5 @@
-if (typeof window !== 'undefined') {
+import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment';
+
+if (ExecutionEnvironment.canUseDOM) {
```

Comment:
Ha, I remember bumping into this discussion! Glad it's resolved 🙂

Reply:
Except that `ExecutionEnvironment.canUseDOM` does exactly the same thing as `typeof window !== 'undefined'` (except doing a few more checks) 🤦‍♂️

It's now officially made its way to the docs, now that client modules are finalized and published: https://docusaurus.io/docs/advanced/client#:~:text=remember%20to%20check%20the%20execution%20environment